src/lib/import-export/import/importers/importer.ts

```diff
@@ -71,9 +71,45 @@ export abstract class BaseImporter extends EventEmitter implements Importer {
 			}
 		}
 
+		this.fixSiteUrl( siteId );
 		this.emit( ImportEvents.IMPORT_DATABASE_COMPLETE );
 	}
 
+	private async fixSiteUrl( siteId: string ) {
```

Comment:
Nitpick: could it be 'update' or 'replace' instead of 'fix'?

Reply:
Done!